### PR TITLE
User: Card number for adjusting of users

### PR DESCRIPTION
**Pull request by davidwessman**
Adding card_number field to user
Adding index with remote form for searching with magnetic card reader
Using inputmask to proper format the field

### CODE REVIEW

File: app/views/users/_form.html.erb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <%= simple_form_for(user, url: own_user_path, html: { class: 'reg-page' }) do |f| %>
   <%= f.input :firstname %>
   <%= f.input :lastname %>
+  <%= f.input :card_number, placeholde: 'Kortnummer', input_html: {data: {mask: '9999-9999-9999-9999'}} %>
```
Comment (navro) [inline app/views/users/_form.html.erb]: Should be placeholder?

File: app/views/admin/adjustments/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<%= provide(:title, t('adjustment.title')) %>
+<div class="headline">
+  <h1><%= t('adjustment.title') %></h1>
+</div>
+
+<div class="col-md-3 col-sm-12">
+  <%= link_to t('vote_user.attendance_list'), attendance_list_admin_vote_users_path,
+              class: 'btn primary' %>
```
Comment (navro) [inline app/views/admin/adjustments/index.html.erb]: I think manage_voting, User is required to access the attendance_list

File: app/views/admin/adjustments/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  <% if @agenda.present? %>
+    <%= @agenda %>
+  <% end %>
+
+  <br>
+
+  <%= simple_form_for(:vote_user, url: search_admin_vote_users_path, remote: true) do |f| %>
+    <%= f.input(:card_number, required: false, autofocus: true,
+                label: User.human_attribute_name(:card_number),
+                input_html: {data: {mask: '9999-9999-9999-9999'}}) %>
```
Comment (navro) [inline app/views/admin/adjustments/index.html.erb]: We should add autocomplete: 'off' in input_html

Comment (davidwessman) [inline app/views/admin/adjustments/index.html.erb]: I changed it to use a `LIKE` search, perhaps it should only be exact?

**Comment by navro** [comment]
> I changed it to use a LIKE search, perhaps it should only be exact?

I meant that we should disable autocomplete to stop the browser from saving form input :)

File: app/views/admin/adjustments/index.html.erb (hunk 3 above)
Comment (davidwessman) [inline app/views/admin/adjustments/index.html.erb]: Yeah, I fixed that - but do you think it is okay that you can show partial values?

Comment (navro) [inline app/views/admin/adjustments/index.html.erb]: > Yeah, I fixed that - but do you think it is okay that you can show partial values?

If we enter all the digits (that is, read the card), do we only get exact matches?

Comment (davidwessman) [inline app/views/admin/adjustments/index.html.erb]: Yes, since all have the same format and are unique.

Comment (navro) [inline app/views/admin/adjustments/index.html.erb]: > Yes, since all have the same format and are unique.

Then i think it's more than OK :smile:

**Comment by navro** [comment]
Looks really nice! +1

**Pull request merged**